crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -6,6 +6,7 @@
 //! - Keys that exist for bookkeeping but don't correspond to the schema have a `_` prefix
 
 use std::collections::BTreeMap;
+use std::collections::BTreeSet;
```

Comment:
Nit: this could be condensed
```suggestion
use std::collections::{BTreeMap, BTreeSet};
```

Reply:
For myself, I prefer not combining them.  I've dealt with many complex `use`s that are annoying to read, edit, and cause conflicts and much prefer each lint being independent.